xdsl/backend/csl/print_csl.py

```diff
@@ -68,8 +71,63 @@ def register_binops(self):
                 arith.RemSI.name: "%",
                 arith.RemUI.name: "%",
                 arith.ShLI.name: "<<",
+                arith.AndI.name: "&",
+                arith.OrI.name: "|",
             }
         )
+        self._cmp_ops.update(
+            {
+                arith.Cmpi.name: {
+                    "eq": "==",
+                    "ne": "!=",
+                    "slt": "<",
+                    "sle": "<=",
+                    "sgt": ">",
+                    "sge": ">=",
+                    "ult": "<",
+                    "ule": "<=",
+                    "ugt": ">",
+                    "uge": ">=",
+                },
+                arith.Cmpf.name: {
+                    "false": None,
+                    "oeq": "==",
+                    "ogt": ">",
+                    "oge": ">=",
+                    "olt": "<",
+                    "ole": "<=",
+                    "one": "!=",
+                    "ord": None,
+                    "ueq": "==",
+                    "ugt": ">",
+                    "uge": ">=",
+                    "ult": "<",
+                    "ule": "<=",
+                    "une": "!=",
+                    "uno": None,
```

Comment:
What happens in this case?

Reply:
There is a check on line 122 for when the op maps to `None`.

Specifically for `ord` and `uno` we throw an error (line 127), because I think for now we don't need to handle this, and if we do, we will find out quickly when the error is thrown.